Util/FormBuilderIterator.php

```diff
@@ -65,12 +65,7 @@ public function __construct(FormBuilderInterface $formBuilder, $prefix = false)
      */
     private static function getKeys(FormBuilderInterface $formBuilder)
     {
-        if (!self::$reflection) {
```

Comment:
I removed the whole reflection part, because sonata access a private property "children". Not every FormBuilderIterator has this property. Its described in my first comment for this PR.